compiler/Cargo.lock

```diff
@@ -1798,9 +1814,9 @@ dependencies = [
 
 [[package]]
 name = "serde"
-version = "1.0.126"
+version = "1.0.136"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "ec7505abeacaec74ae4778d9d9328fe5a5d04253220a85c4ee022239fc996d03"
+checksum = "ce31e24b01e1e524df96f1c2fdd054405f8d7376249a5110886fb4b658484789"
```

Comment:
I'm surprised to see all these version changes. Are these expected for some reason?

Reply:
Hmmm, this is unintentional but I can't figure out how to make this not happen. Any tips here? 

Like there's no way for me to add `intern` as a dep to this crate without those other ones updating.

Reply:
Strange. I think it's probably fine if it's happening automatically.